Source/Prism/Prism.csproj

```diff
@@ -1,35 +1,46 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.0;</TargetFrameworks>
+    <TargetFrameworks>netstandard1.0;netstandard2.0;net45;uap10.0</TargetFrameworks>
```

Comment:
Not sure if this won't give an issue (both 1.0 and 2.0 if in the same package). I remember Laurent saying he needed to create 2 different NuGet packages for it to work because the consuming project didn't know which one to use. But I might confuse targetting and compiling vs consuming.

Reply:
shouldn't create an issue at all... if anything it is better because it provides backwards compatibility for existing PCL projects that need netstandard1.0 while it removes the requirement for `System.Resources.ResourceManager` for projects targeting net45, uwp, or netstandard2.0